packages/errors/src/vars/index.ts

```diff
@@ -0,0 +1,45 @@
+import { keyMirror } from '@pluralsight/ps-design-system-util'
+
+const illustrations = ['error'] as const
+
+export type IllustrationNames = typeof illustrations[number]
+
+export const illustrationNames = keyMirror(...illustrations)
+
+export const codes = {
+  403: {
+    illustration: illustrationNames.error,
+    heading: 'You’re not authorized to view this page',
+    caption: 'Try contacting support or your plan administrator.',
```

Comment:
The visuals have changed here. Please restore the old visual design where there is no visual  distinction between heading and caption.

Reply:
@jaketrent Take a look at the [designs](https://www.figma.com/file/YsTklBecdddy9RZ3HXddIseD/PSDS-Web-Components?node-id=11293%3A29948). There is now a distinction between heading and caption.

Reply:
Again, good to know.  I did not know this. Thanks.